tasks/node/Makefile

```diff
@@ -9,3 +9,7 @@ test::
 .PHONY: build
 build::
 	@npm run build
+
+.PHONY: publish
+publish::
```

Comment:
Can we add a trailing line feed so we don't get the angry red "missing EOF line feed" here?

We don't have any pre-commit stuff set up for these LCAF repos to enforce this, but we probably ought to look into that.